include/flatbuffers/flatbuffers.h

```diff
@@ -1470,6 +1502,21 @@ class FlatBufferBuilder
     finished = true;
   }
 
+  // Allocates space for a vector of structures.
+  // Must be completed with EndVectorOfStructs().
+  template<typename T>
```

Comment:
Correct, the existing use cases are in violation of Google style.  Rather than proliferate the problem I've added code that adheres to Google style.  The idea is that a follow up can consist of a pass to align style so that we can use auto formatting tools like clang-format to keep things in check.